tests/execution_spec.rs

```diff
@@ -254,6 +255,7 @@ enum ConfigSource {
 struct Mock {
     request: UpstreamRequest,
     response: UpstreamResponse,
+    assert_n: Option<usize>,
```

Comment:
We are expecting `n` hits for this mock. Let's say that and add bit of documentation here about each field. 
```suggestion
    expect_hits: usize,
```
Set the default value to `1`.